docs/presentation/Content/DefaultPropertyCategoryOverride.md

```diff
@@ -8,26 +8,44 @@ The default property category is a category that gets assigned to properties tha
 
 ## Attributes
 
-| Name                 | Required? | Type                                                                  | Default | Meaning                                                                                  |
-| -------------------- | --------- | --------------------------------------------------------------------- | ------- | ---------------------------------------------------------------------------------------- |
-| *Picking attributes* |
-| `requiredSchemas`    | No        | [`RequiredSchemaSpecification[]`](../Advanced/SchemaRequirements.md)  | `[]`    | Specifications that define schema requirements for the rule to take effect.              |
-| `priority`           | No        | `number`                                                              | `1000`  | Defines the order in which presentation rules are evaluated.                             |
-| `onlyIfNotHandled`   | No        | `boolean`                                                             | `false` | Should this rule be ignored if there is already an existing rule with a higher priority. |
-| *Content Modifiers*  |
-| `specification`      | Yes       | [`PropertyCategorySpecification`](./PropertyCategorySpecification.md) |         | Specification for the custom property category                                           |
-
-## Example
-
-```JSON
-{
-  "ruleType": "DefaultPropertyCategoryOverride",
-  "requiredSchemas": [{ "name": "BisCore", "minVersion": "1.0.1" }],
-  "priority": 9999,
-  "onlyIfNotHandled": true,
-  "specification": {
-    "id": "default",
-    "label": "General Properties"
-  }
-}
+| Name                                            | Required? | Type                                                                  | Default |
```

Comment:
Missing `onlyIfNotHandled` attribute

Reply:
I intentionally skipped it, because it doesn't make sense for `DefaultPropertyCategoryOverride` - we always handle just one rule of this type, so setting `onlyIfNotHandled` doesn't change anything.